.github/workflows/publish.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       matrix:
           os: ['ubuntu-20.04']
           python-version: ['3.8', '3.9', '3.10', '3.11']
-          pytorch-version: ['2.1.2']  # Must be the most recent version that meets requirements.txt.
+          pytorch-version: ['2.2.1']  # Must be the most recent version that meets requirements.txt.
```

Comment:
Can we upgrade torch 2.2.2 instead?

Reply:
Technically we can, but I'm afraid it's too new and has not been verified by the community yet (e.g. maybe some new bugs). Let's wait for some time (say one month) to decide if we want to move to pytorch 2.2.2 ?

Reply:
I see. Makes sense!